oam/install_scripts/mcs-savebrm.py.in

```diff
@@ -40,10 +40,15 @@ if __name__ == '__main__':
     is_primary = False
 
     # For multi-node with local storage or default installations
-    if (storage.lower() != 's3' and master_addr != default_addr) or \
-master_addr == default_addr:
+    multi_with_local_storage = storage.lower() != 's3' and master_addr != default_addr
+    if multi_with_local_storage is True or master_addr == default_addr:
+        # This might be not a primary in a multi-node cluster but we are safe 
+        # to save BRM locally
         is_primary = True
```

Comment:
This code path will also execute in a multinode w/ local storage and sharing dbroots, but I do not believe save brm will cause issue across a shared dbroot, as this was previously working and your changes are merely aesthetic 👍